src/main/java/com/fouristhenumber/utilitiesinexcess/common/items/ItemInversionSigilActive.java

```diff
@@ -2,29 +2,527 @@
+        } else if (world.provider.dimensionId == 0) {
+            player.addChatMessage(
+                new ChatComponentText(StatCollector.translateToLocal("chat.pseudo_inversion_ritual.overworld")));
+        } else if (world.provider.dimensionId == -1) {
+            player.addChatMessage(
+                new ChatComponentText(StatCollector.translateToLocal("chat.pseudo_inversion_ritual.nether")));
+        } else {
+            player.addChatMessage(
+                new ChatComponentText(StatCollector.translateToLocal("chat.pseudo_inversion_ritual.misc")));
+        }
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestNorthContents",
+                    (chestNorthContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestEastContents",
+                    (chestEastContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestSouthContents",
+                    (chestSouthContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector.translateToLocalFormatted(
+                    "chat.pseudo_inversion_ritual.chestWestContents",
+                    (chestWestContentsOk ? "✓" : "✗"))));
+        player.addChatMessage(
+            new ChatComponentText(
+                StatCollector
+                    .translateToLocalFormatted("chat.pseudo_inversion_ritual.spiral", (spiralOk ? "✓" : "✗"))));
+        if (dimensionOk && chestNorthContentsOk && chestEastContentsOk && chestSouthContentsOk && chestWestContentsOk) {
+            player.addChatMessage(new ChatComponentTranslation("chat.pseudo_inversion_ritual.ready"));
+        }
+        return true;
+    }
+
+    @SubscribeEvent(priority = EventPriority.NORMAL)
+    public void whenServerTick(TickEvent.ServerTickEvent event) {
+
+        List<EntityPlayer> playerList = getSiegePlayers();
+        for (EntityPlayer player : playerList) {
+            EntitySiegeProperty properties = getProperties(player);
+            if (properties.siege) {
+                properties.siegeTimer--;
+            }
+            if (properties.siegeTimer <= 0 && properties.siege) {
+                properties.siegeTimer = 60 + player.getEntityWorld().rand.nextInt(41);
+                EntityMob entitymob = null;
+                int mobType = player.getEntityWorld().rand.nextInt(4);
+                switch (mobType) {
+                    case 0:
+                        entitymob = new EntityZombie(player.getEntityWorld());
```

Comment:
I remember than in ExtraUtilities you could get EntityGiantZombie with a rare chance

Reply:
yeah, EntityGiantZombie is a minecraft class that exists, would be cool if you add a small chance of it spawning

i think adding 1/50 or 1/25 chance is enough (zombie is already 1/4 so giant zombie would be 1/4 * 1/25 = 1/100), but you can as well ask in the chat if it's a cool addition for other opinions